pages/linux/qm-template.md

```diff
@@ -0,0 +1,8 @@
+# qm template
+
+> Create a Template.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Create a template of a specific virtual machine:
+
+`qm template {{vmid}}`
```

Comment:
```suggestion
`qm template {{vm_id}}`
```
Thanks for your contribution. I have a suggestion for this page.

Reply:
I didn't find any guidelines on how to name placeholder variables, that's why I chose to follow the original documentation.